packages/@ama-sdk/core/src/fwk/mocks/base-mock-adapter.ts

```diff
@@ -45,6 +41,27 @@ export abstract class BaseMockAdapter implements MockAdapter {
     return id;
   }
 
+  /**
+   * @inheritDoc
+   * @deprecated Please use getOperationIdAsync instead
+   */
+  public getOperationId(request: EncodedApiRequest): string {
+    if (typeof this.pathObjects === 'function') {
+      throw new Error(`The pathObjects argument must be of type PathObject[]`);
+    }
+    const object = getPath(request.basePath, this.pathObjects, request.method);
+    return this.extractOperationIdFromPath(request, object);
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public async getOperationIdAsync(request: EncodedApiRequest): Promise<string> {
```

Comment:
Deprecating `getOperationId` means that it will be removed in favor of `getOperationIdAsync`, not sure the suffix `Async` will still make sense when `getOperationId` will be removed.
Maybe you can name it `retrieveOperationId` (or a more generic name)

Reply:
Renamed it to `retrieveOperationId`